lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -1040,7 +1033,16 @@ def cast_to_string(string: str) -> Optional[str]:
 
     def cast_to_integer(string: str) -> Optional[int]:
         try:
-            return int(string)
+            value = int(string)
+            # Age Ceiling
+            if question_id == 'age':
```

Comment:
It is a bit unusual to special case age and age_months inside the generic cast_to_integer function. Instead, we could have created `age_months_ceiling` or something like that.
But given that the cast_to_integer function is in this specific ETL code and is not in a library shared across ETLs, that's OK.

Reply:
Hmm. Now that you've pointed it out, it does seem like a weird side-effect for a function called `cast_to_integer`. I can move the age_ceiling outside of the cast function, but still stay within `questionnaire_item`.

Reply:
Made this change in the latest commit.

Reply:
Right on!